internal/handler/handler.go

```diff
@@ -0,0 +1,109 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Arcadian-Sky/musthave-metrics/internal/storage"

```

Comment:
CHANGE REQUEST: please format imports

https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package

Reply:
this is grouping by vscode,it already makes grouping by types
Do you mean  separate local package from others like this?
```
import (
	"fmt"
	"net/http"

	"github.com/Arcadian-Sky/musthave-metrics/internal/storage"

	"github.com/go-chi/chi/v5"
)
```

Unfortunatelly, I don't understand how to make local repository name shorter, for example by alias
And is there any way to make an automatically separation 'goimports -local' doesn't work